tests/test_api.py

```diff
@@ -33,6 +33,14 @@ class TestAPI:
     and header checks
     """
 
+    @pytest.mark.parametrize('endpoint', ENDPOINTS)
+    def test_no_content_type(self, client, endpoint):
+        """ Test that no 500 is thrown when the content isnt specified """
+        resp = client.post(endpoint, data='{}')
+        assert resp.status_code < 500
+        resp = client.patch(endpoint, data='{}')
+        assert resp.status_code < 500
+
     @pytest.mark.parametrize('endpoint,method,status_code',
                              [(ept, 'GET', 200) for ept in ENDPOINTS] +
                              [(ept+'/123', 'GET', 404) for ept in ENDPOINTS]
```

Comment:
Maybe also a test to ensure that mime-type isn't required would be good. In case someone creates a new resource and forgets to use request.get_json(force=True) for some reason. We'd catch that in tests

Reply:
That is tested here (no headers in the post or patch)

Reply:
👍